repose-aggregator/components/filters/uri-stripper-filter/src/main/scala/org/openrepose/filters/uristripper/UriStripperFilter.scala

```diff
@@ -231,6 +252,20 @@ class UriStripperFilter @Inject()(configurationService: ConfigurationService)
     }
   }
 
+  private def transformXmlLink(tryResponseXml: Try[XmlValue], linkPath: LinkPath, strippedToken: String, previousToken: Option[String], nextToken: Option[String]): Try[XmlValue] = {
+    val transformerFactory = TransformerFactory.newInstance()
+    val xsltTransformer = transformerFactory.newTransformer(xsltSource)
+    xsltTransformer.setParameter(parameterName, parameterValue)
+
+    tryResponseXml map { responseXml =>
+      Try(responseXml.transform)
+
+
+
+
```

Comment:
yeah.. that was where i finished friday so i pushed, in retrospect i should have just stashed the unfinished changes.